runtime/src/accounts_db.rs

```diff
@@ -981,6 +1096,14 @@ impl AccountsDB {
                                     );
                                 }
 
+                                let has_zero_lamport_accounts = locked_entry
```

Comment:
nits: carry over this comment? https://github.com/solana-labs/solana/pull/14592/files#r562004860

Yeah, what a burden to maintain 2 branches... Thanks for doing so.